wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -321,7 +322,8 @@ instance Buildable (SecureLog a) => Buildable (SecureLog (V1 a)) where
 instance (Buildable a, Buildable b) => Buildable (a, b) where
     build (a, b) = bprint ("("%build%", "%build%")") a b
 
-
+instance Buildable Version where
+    build v = bprint shown v
```

Comment:
:thinking:  Where does that come from? Why using the underlying `Show` instance here, that feels wrong.
Plus, it's another wild orphan as we are supposed to removed them :sweat_smile:

Reply:
I got it from:
```
wallet/src/Pos/Wallet/Web/ClientTypes/Types.hs
675:instance Buildable Version where
```

Reply:
`shown` is from `Formatting(shown)`

Reply:
Yup, this I understand ^.^ ... My question was more like: Why do we need this instance in a first place? Where is this `Version` used? 

If it's only in:


```hs
-- | The @static@ settings for this wallet node. In particular, we could group
-- here protocol-related settings like the slot duration, the transaction max size,
-- the current software version running on the node, etc.
data NodeSettings = NodeSettings {
     setSlotDuration   :: !SlotDuration
   , setSoftwareInfo   :: !(V1 Core.SoftwareVersion)
   , setProjectVersion :: !Version
   , setGitRevision    :: !Text
   } deriving (Show, Eq, Generic)
```

I'd rather stick a `V1 Version` here and define a non-orphan instance for `V1 Version`. This shouldn't break much thing and following the compiler steps will be easy.